ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline-serial.yml

```diff
@@ -38,6 +38,9 @@ spec:
     - name: submit
       description: Set to "true" to submit results and open a pull request.
       default: "false"
+    - name: image_tag
```

Comment:
Can we rename this, and the task params, to `pipeline_image` instead? It's not just a tag nor does it need to contain one. It can be any valid pull spec.